app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -128,14 +129,6 @@ public boolean runAction(String action) {
                 });
                 return true;
             }
-            case ACTION_MOVE_UP: {
-                moveLineBy1(true);
-                return true;
-            }
-            case ACTION_MOVE_DOWN: {
-                moveLineBy1(false);
-                return true;
-            }
```

Comment:
Moved into TextActions. Eventually, I would like to get rid of `CommonTextActions`. Imo it is redundant with the `TextActions` base class